openpype/hosts/blender/utility_scripts/is_workfile_out_of_date.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+from datetime import datetime
+
+from openpype.lib.dateutils import get_timestamp
+from openpype.client.entities import get_last_version_by_subset_name
+from openpype.pipeline import legacy_io
+from openpype.hosts.blender.api.workio import current_file
+
+
+def is_work_file_out_of_date() -> bool:
```

Comment:
This function doesn't belong here. It'd be more appropriate in `blender/pipeline.py`. `utility_scripts` is made for scripts to be run in blender using the `-P` CLI attribute.